### PR TITLE
Added projects page with styling

### PR DESCRIPTION
**Pull request by cyu6**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -35,4 +35,18 @@
 #homelinks img {
     width: 50px;
     padding: 10px;
+}
+
+#projectscontent {
```
Comment (paulaksp) [inline portfolio/src/main/webapp/style.css]: formatting, use two spaces for css generally preferred than four.

File: portfolio/src/main/webapp/projects.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+        <div class="projectblock" id="cdpush">
+            <h3>Config Cleanup @ Google</h3>
+            <h4>June - September 2019 | Engineering Practicum Intern</h4> 
+            <p>CDPush team in Sunnyvale. Developed a C++ server to scan and clean up files in a storage system.</p>
+            <p>Used C++, learned about Go, distributed systems, file management systems</p> 
+        </div>
+        <h2>I've recently become interested in UI/UX design:</h2>
+        <ul>
+            <li>MIT Beat the Pandemic II Wireframe:</li>
```
Comment (paulaksp) [inline portfolio/src/main/webapp/projects.html]: nit: is there additional things after the ':' that is meant to be there?

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  <body>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div>
```
Comment (paulaksp) [inline portfolio/src/main/webapp/projects.html]: overkill comment for this starter project, but just as an fyi
-- navbar isn't reusable even though i assume you'll be using the same div in all the various pages. so changing the navbar requires changing each of this replica in all the pages.

later for the capstone or more complicated front-end, having some form of templating (either through importing the HTML, or through some framework like React, etc).

again, no need to act on this but just a random note.

**Review by paulaksp — CHANGES_REQUESTED** [review]
Include screenshot(s) for any UI related PRs.

**Comment by cyu6** [comment]
projects

File: portfolio/src/main/webapp/projects.html (hunk 2 above)
Comment (cyu6) [reply portfolio/src/main/webapp/projects.html]: Yup I just didn't look up the links for what I wanted to attach yet so they're like placeholders for now

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -33,6 +33,20 @@
 }
 
 #homelinks img {
-    width: 50px;
-    padding: 10px;
+  width: 50px;
+  padding: 10px;
+}
+
+#projectscontent {
```
Comment (paulaksp) [inline portfolio/src/main/webapp/style.css]: nit: use dashes/hypens to concatenate words for css selectors
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -33,6 +33,20 @@
...
+  margin-right: auto;
+  left: 10%;
+  position: relative;
+}
+#projectscontent .projectblock {
+  border: 1mm;
+  border-color: lightcoral;
```
Comment (paulaksp) [inline portfolio/src/main/webapp/style.css]: nit: generally prefer hex colors compared to the name colors (ie lightcoral = `#f08080`), assuming that there will be more colors being used across codebase, some of the hex-colors not having a corresponding name.

**Review by paulaksp — APPROVED** [review]
Generally LGTM.

Some of the css might need to change in future PRs depending on whether you want things to be responsive, or how you'd like things to look at with various screen-sizes, etc but we can discuss the styling part of it separate from the PR depending on your requirements and thresholds for how to get things to look a certain way.

**Comment by cyu6** [comment]
The main commit is ae5a74f - I tried rebasing on another branch and ended up duplicating the commits (oops)

**Review by paulaksp — APPROVED** [review]
Wasn't really sure what you meant by the duplicating commits, but LGTM.

**Pull request merged**